createDockerCommands.py

```diff
@@ -60,11 +81,11 @@
   not_executed_algos = set()
 
   for i in range(startIteration, 1+stopIteration):
-    print(analysis + ' ' + datasetID + ' ' + classVar + ' ' + 'iteration' + str(i))
-    path = '/Analysis_Results/' + analysis + '/' + datasetID + '/' + classVar + '/iteration' + str(i) + '/*/' + outFileToCheck
-
+    print(f'{analysis} {datasetID} {classVar} iteration {str(i)}')
```

Comment:
Solid use of `f""` strings and `os.path.join()` to make these lines a ton shorter!